README.md

```diff
@@ -38,34 +38,35 @@ If you would like to translate the game into another language check the #contrib
 Refer to [the Space Wizards' guide](https://docs.spacestation14.com/en/general-development/setup/setting-up-a-development-environment.html) on setting up a development environment for general information, but keep in mind that Einstein Engines is not the same and many things may not apply.
 We provide some scripts shown below to make the job easier.
 
-### Build dependencies
+### Build Dependencies
 
-> - Git
-> - DOTNET SDK 7.0 or higher
-> - python 3.7 or higher
+> - DOTNET SDK 8.0.100 or similar
 
+### Recommended Tools
+
+> - Git (needed)
+> - Python 3.7 or higher
```

Comment:
One of the tests tries to update submodules using the RUN_THIS.py. It would fail if python isn't installed.

Reply:
We need to just nuke the RUN_THIS.